src/components/Label.tsx

```diff
@@ -6,15 +6,18 @@ interface LabelProps {
   // We don't usually have `fooProps`-style props, but this is for/from react-aria
   labelProps?: LabelHTMLAttributes<HTMLLabelElement>;
   label: string;
+  suffix?: string;
```

Comment:
For doing `label=First Name`, `suffix=Required` the idea being we don't want "required" to leak into the react-aria label directly. I assume they have like aria-required attributes? Not sure, but just seems good to keep separate.